bin/twlight_unwaitlist.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+#
+# Un-waitlists proxy partners having atleast one inactive authorization. Doesn't work for partners with streams.
+
+# Environment variables may not be loaded under all conditions.
+if [ -z "${TWLIGHT_HOME}" ]
+then
+    source /etc/environment
+fi
+
+# Load virtual environment
+if source ${TWLIGHT_HOME}/bin/virtualenv_activate.sh
+then
+    echo "Probing for waitlisted partners with atleast one active authorization"
```

Comment:
Could you reword this? It's a little unclear. We're looking for partners with at least one available account, not one active authorization, right?